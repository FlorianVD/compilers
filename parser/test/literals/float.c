int literals()
{
    float y = 42.5;
}
