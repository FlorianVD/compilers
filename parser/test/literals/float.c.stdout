└── Program
    └── FuncDecl: returnType = 'int', name = 'literals'
        └── CompoundStmt
            └── VarDecl: type = 'float', name = 'y'
                └── FloatLiteral: value = '42.5'
